core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -511,6 +516,8 @@ object ConfigCommand extends Config {
       .ofType(classOf[String])
     val alterOpt = parser.accepts("alter", "Alter the configuration for the entity.")
     val describeOpt = parser.accepts("describe", "List configs for the given entity.")
+    val allOpt = parser.accepts("all", "List all configs for the given entity")
```

Comment:
The KIP proposed adding a `--dynamic` argument as well to allow the user to explicitly specify dynamic configurations. My thought was that one day we might want to change the default behavior to be --all. We don't necessarily have to implement this if it doesn't make sense. What do you think?

By the way, we may want to mention in the doc that this option only differs from the normal output when a specific broker is targeted.

Reply:
Thanks @hachikuji for the comment! Quickly:
* as `--dynamic` is equivalent to the absence of `--all`, IMO it's ok to defer introducing `--dynamic` until a concrete use case arise, but please let me know if you feel otherwise.
* 100% agreed we need to doc this via documentation. based on previous feedback, i have kept the in-code-doc somewhat generic to avoid churn, but i've added additional validation logic in `checkArgs` to make sure that an appropriate exception is raised with relevant error message. Let me know what you think.

Reply:
@raymondng I think I'd prefer not to make this a strict validation. There is no semantic problem with using `--all` if you are describing topic configs; it's just that it's equivalent to the default behavior. Instead could we just add some extra detail to the description? I was thinking something like this:
> List all configs for the given entity (includes static configuration when the entity type is `brokers`)

Reply:
Sounds good! Will incorporate.